src/Resolvers/DependencyPatches.php

```diff
@@ -27,18 +25,24 @@ public function resolve(PatchCollection $collection, PackageEvent $event)
 
         $operations = $event->getOperations();
         foreach ($operations as $operation) {
-            if ($operation->getJobType() == 'install' || $operation->getJobType() == 'update') {
-                // @TODO handle exception.
-                $package = $this->getPackageFromOperation($operation);
-                /** @var PackageInterface $extra */
-                $extra = $package->getExtra();
-                if (isset($extra['patches'])) {
-                    $patches = $this->findPatchesInJson($extra['patches']);
-                    foreach ($patches as $package => $patch_list) {
```

Comment:
While we didn't use the `$package` var here, I think this style provided some clarity around the data structure. Can you revert back to this notation?

Reply:
https://www.facebook.com/groups/millennialshow/permalink/2074484859516440/

Reply:
(just the `foreach` line - the rest of the style changes are fine)

Reply:
I updated it, changed the variable into something more meaningful without wasting memory with unused vars.